tests/integration-tests/tests/schedulers/conftest.py

```diff
@@ -104,8 +102,8 @@ def _create_database_stack(stack_factory, request, region, vpc_stack_for_databas
             {"ParameterKey": "ClusterName", "ParameterValue": cluster_name},
             {"ParameterKey": "Vpc", "ParameterValue": vpc_stack_for_database.cfn_outputs["VpcId"]},
             {"ParameterKey": "AdminPasswordSecretString", "ParameterValue": admin_password},
-            {"ParameterKey": "Subnet1CidrBlock", "ParameterValue": "192.168.8.0/23"},
-            {"ParameterKey": "Subnet2CidrBlock", "ParameterValue": "192.168.4.0/23"},
+            {"ParameterKey": "Subnet1CidrBlock", "ParameterValue": CIDR_FOR_CUSTOM_SUBNETS[-1]},
+            {"ParameterKey": "Subnet2CidrBlock", "ParameterValue": CIDR_FOR_CUSTOM_SUBNETS[-2]},
```

Comment:
Out of curiosity ... why are you picking CIDRs in reverse order ?

Reply:
quick trick to make sure they aren't already taken

Reply:
mmm ... how long it will hold ?
seems risky

Reply:
tests only use first element of CIDR_FOR_CUSTOM_SUBNETS
In the next refactoring I think we may want to drop CIDR_FOR_CUSTOM_SUBNETS completely and rather of using a subnet in same AZ different CIDR, we may want to use a different subnet in different AZ